components/brave_welcome_ui/theme/welcome-theme.ts

```diff
@@ -0,0 +1,9 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+export default interface IThemeWelcomePage {
+  color: {
+    outlineColor: string
```

Comment:
This seems like something that would be useful for all brave components. Is there a reason outline color would be something that welcome page would have only?

Reply:
I agree it would be useful for all brave features but adding this global default here seems outside of this PR's scope. I'm happy to do that in a follow-up task where we move our brave-core themes to this repo

Reply:
I don't think it's outside of this PR scope since this is the first PR that's adding this value. Either way requires a new brave-ui version - allowing for separate layer of themes (which you've discovered is problematic and requires using the drastic `any` step, thereby removing all type checking for theme props), or simply adding this new property.